calcom/cal.com
# Cal fixes andrea

## What does this PR do?



Fixes #2149
I had change the useEffect() in DestinationCalendarSelector.tsx, if selectedOption and selected do not exist so defaultValue will be the first calendar you integrate.
<img width="460" alt="image" src="https://user-images.githubusercontent.com/19562383/159814733-052cf077-d0f2-49d9-a29a-20d4ba751055.png">


## Type of change




## How should this be tested?



Run the app and then go to Integrations and you are going to see the first calendar you integrate.
